EIPS/eip-7637.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 7637
+title: EXTCODEHASH optimize
+description: Modify the output value of a situation in EXTCODEHASH
```

Comment:
description needs to be more specific instead of mentioning "a situation"
```suggestion
description: Modify the output value of EXTCODEHASH for EOA accounts to `0x`
```